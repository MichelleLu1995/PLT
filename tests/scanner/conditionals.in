if
else
elif
for
while
