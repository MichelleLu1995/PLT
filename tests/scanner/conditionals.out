IF
ELSE
ELIF
FOR
WHILE
